db/column_family.cc

```diff
@@ -1302,7 +1293,7 @@ void ColumnFamilyData::InstallSuperVersion(
           new_superversion->write_stall_condition, GetName(), ioptions());
     }
     if (old_superversion->Unref()) {
-      old_superversion->Cleanup(noImmMemoryContribution);
```

Comment:
Remove changes made during #8454 when the mempurge process was happening in the middle of the SwitchMemtable process.